builtins/amp-ad.js

```diff
@@ -199,17 +200,18 @@ export function installAd(win) {
     isPositionFixed() {
       let el = this.element;
       const body = el.ownerDocument.body;
+      let hasFixedAncestor = false;
       do {
         if (POSITION_FIXED_TAG_WHITELIST[el.tagName]) {
           return false;
         }
         if (this.getWin()/*because only called from onLayoutMeasure */
             ./*OK*/getComputedStyle(el).position == 'fixed') {
-          return true;
+          hasFixedAncestor =  true;
```

Comment:
might be good to document this , as the reason for this cant be figured out just by reading the code.